### ISSUE TITLE
Add crossOrigin="anonymous" on the script tag to load gpt.js blocked by browser

### ISSUE THREAD

**Issue by you1cheng** [googleads/google-publisher-tag-samples#43]
You can check this demo:
https://jsfiddle.net/r7khofp9/5/

and open chrome devtools, there is an error in the console:
`Access to script at 'https://www.googletagservices.com/tag/js/gpt.js' from origin 'https://fiddle.jshell.net' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.`

I think the reason there is no `Access-Control-Allow-Origin` in the response header

**Comment by jimper** [comment]
Hi @you1cheng,

This is expected, GPT does not support loading with `crossorigin="anonymous"`. However, GPT can be requested from a special [limited ads URL](https://developers.google.com/publisher-tag/guides/general-best-practices#load_from_an_official_source), which is a cookie-less domain used for regulatory compliance. If that's not a suitable alternative for your use case, I recommend reaching out to [Ad Manager support](https://support.google.com/admanager/community) for further assistance, since this isn't directly related to the official GPT samples (and is out of scope for this support channel).
